src/components/sign-in/sign-in.tsx

```diff
@@ -147,8 +147,10 @@ export const connectWallet = (setChainData: SettableChainData['setChainData']) =
   if (connectionError || !web3ModalProvider) return;
 
   // Wrapped in callback to prevent synchronous error, because `request` property is not guaranteed to exist
-  // TODO: Should we call this in case of wallet connect?
-  const [requestAccountsError] = await go(() => web3ModalProvider.request({ method: 'eth_requestAccounts' }));
+  const [requestAccountsError] = await go(() => {
+    if (web3ModalProvider.isMetaMask) return web3ModalProvider.request({ method: 'eth_requestAccounts' });
```

Comment:
@andreogle @bbenligiray I've noticed that this was causing an issue for wallet connect.

I am kinda sad, that I look at the issue sooner, because it was probably our (my) mistake from the start and not just wallet connect flakiness. Though, I am not sure why this suddenly stopped working as it previously did work (when we tested with Santiago)...

Anyway, It's still not 100% reliable though (even with 1.6 update)